### DISCUSSION TITLE
AsusFX504 - 0.7.3

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#7] [General]
### News / Fixes:
- Updated PNLF SSDT
- Updated all acidanthera kexts and boot files to latest versions.
- Updated the config.plist.
- Now maybe you can control fans (to be tested...)

### Old Know Issues:
- HDMI have issues with some resolutions.

---

**RELEASE Version is recommended.**

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/releases/tag/7.0'>AsusFX504 - 0.7.3</a>.</em>

**Comment by jxonitsway** [comment]
Nice work, I had been working with my Big Sur 11.6 well, but how's controlling fan speed work? Any software or application you're using?

**Reply by RobyRew** [reply]
its not a possible thing yet, i would be looking at some time to see if i do something

https://github.com/hieplpvip/AsusSMC

**Reply by RobyRew** [reply]
> Nice work, I had been working with my Big Sur 11.6 well, but how's controlling fan speed work? Any software or application you're using?

BTW: I just updated EFI to OpeCore 0.7.4 with some nice fixes ;)
